client/src/context/useKanbanContext.tsx

```diff
@@ -49,18 +49,21 @@ export const KanbanProvider: FunctionComponent = ({ children }): JSX.Element =>
 
     const { destination, source, draggableId } = result;
 
+    const dupBoard = Object.assign({}, activeBoard);
```

Comment:
Not a critique, but can you explain what you're doing here with Object.assign?

Reply:
Yep! I don't want to manipulate `activeBoard` itself so I'm duplicating it, manipulating, then setting to activeBoard